assets/js/components/wp-dashboard/WPDashboardWidgets.stories.js

```diff
@@ -77,7 +78,7 @@ ReadyWithActivateModuleCTA.args = {
 		registry.dispatch( CORE_SITE ).receiveSiteInfo( {
 			adminURL: 'http://example.com/wp-admin/',
 		} );
-		// setupSearchConsoleAnalytics4MockReports( registry );
+		setupSearchConsoleMockReports();
```

Comment:
Switched to use only search console mocked data, needed for the 2 search console widgets